src/Drivers/ObjectDriver.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Spatie\Snapshots\Drivers;
 
+use Composer\InstalledVersions;
```

Comment:
Do we need an explicit dependency in `composer.json` for this? 🤔

According to https://getcomposer.org/doc/07-runtime.md#installed-versions this is "only" available for composer 2+. "Shouldn't" be an issue, but who knows.

On that page it can also be seen they add features there which require ^2.1 etc., so I think `composer-runtime-api` needs to be added? 🤔

Reply:
done :)